calico_cni/calico_cni.py

```diff
@@ -512,11 +512,21 @@ def _get_policy_driver(self):
                 print_cni_error(ERR_CODE_GENERIC, e.message)
                 sys.exit(ERR_CODE_GENERIC)
         else:
-            _log.debug("Using Default Kubernetes Policy Driver")
-            driver = policy_drivers.KubernetesDefaultPolicyDriver(
-                    self.network_name
-            )
-
+            # Decide which Kubernetes driver to use.
+            policy_block = self.network_config.get(POLICY_KEY, {})
+            policy_type = policy_block.get("type")
+            if policy_type == POLICY_MODE_ANNOTATIONS: 
```

Comment:
Will there be any log at info level to indicate which policy engine was used? Should there be?